traits/has_traits.py

```diff
@@ -54,6 +54,7 @@
 )
 from .trait_errors import TraitError
 from .util.deprecated import deprecated
+from .util._traitsui_helpers import check_traitsui_major_version
```

Comment:
The module has a private name so (1) it does not get picked up by the API reference and (2) obvious to developers that this can be modified/removed in the future without much worry whether this gets used by other projects.